### PR TITLE
Continuous integration

### LINKED ISSUE #19: how to do continuous integration with lambda functions
@nelsonic I would be happy to work with you on a follow up tutorial to cover this issue. there is no good documentation on it.

### PR DESCRIPTION
**Pull request by nikhilaravi**
Adds a walkthrough of setting up basic Continuous Integration with Codeship, including creating a User and Group, and a custom deployment script. 

fixes #19

### CODE REVIEW

**Comment by nikhilaravi** [comment]
@benjaminlees would be great to get your feedback too! Planning to add another example using a gulp script to automate the archiving and deployment but thought that would be too complicated to begin with.

File: package.json [hunk 1]
```diff
@@ -24,5 +24,9 @@
   "devDependencies": {
     "aws-lambda-mock-context": "^1.1.0",
     "tape": "^4.4.0"
+  },
+  "dependencies": {
+    "babel-core": "^6.4.5",
```
Comment (nelsonic) [inline package.json:29]: They don't appear to be using Babel/ES6 ... do you have access to the "Canary" project? (see gitter chat)

**Comment by nelsonic** [comment]
Looks good. Prefer not to use ES6 in tests unless _absolutely required_.
Merging as I have updated the tests already in #31

**Comment by nikhilaravi** [comment]
@nelsonic sure thats fine! Just wondering, whats the objection to babel?

**Pull request merged**